BUILD.md

```diff
@@ -283,7 +283,7 @@ It fixes some source files to add missing `#include`s.
    Single-config generators:
 
    ```
-   cmake --build .
+   cmake --build . -j $(nproc)
```

Comment:
Seems fine. 

This sets the max of the underlying build tool. Unix Makefiles will default to 1 but Ninja max your system out.